score/score.py

```diff
@@ -0,0 +1,255 @@
+from collections import Counter, namedtuple
+
+from nameinternal import get_card_metadata, get_score_bonus
+from save import Savefile
+
+class Score:
+    def __init__(self, name: str, count: int, should_show: bool):
```

Comment:
Now that I think about it -- should we pass the score directly in the constructor, rather than assigning it later? It essentially amounts to the same, just fewer lines of code